packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -48,24 +75,21 @@ const start = async (...args: string[]) => {
       }
       throw err;
     });
-  }
-  if (args[0] === 'plugin') {
+  } if (args[0] === 'plugin') {
     type = WebpackType.JavaScript;
-  }
-
-  const index = args.indexOf('--name');
-
-  if (index !== -1) {
-    const config = readJsonFile<CLIFlexConfiguration>(paths().cli.pluginsJsonPath);
-    const plugin = config.plugins.find((p) => p.name === args[index + 1]);
+    const plugin = userInputPlugins.filter(p => !p.remote)[0];
+    const config = readPluginsJson();
+    const pluginFind = config.plugins.find((p) => p.name === plugin.name);
 
-    if (!plugin) {
-      throw new FlexPluginError('No plugin file was found with the given name');
+    if (pluginFind) {
+      _updatePluginPort(port, pluginFind.name);
+      setCwd(pluginFind.dir);
     }
-    setCwd(plugin.dir);
   }
 
-  _startDevServer(port, type);
+  const includeAllRemote = args.includes('--include-remote');
```

Comment:
Might as well do this inline now: `_startDevServer(port, userInputPlugins, type, args.includes('--include-remote'));`